Author: Jan Wagner <waja@cyconet.org>
Description:
 Patches check_phpfpm_status to use debian specific paths.

diff --git a/check_phpfpm_status/check_phpfpm_status b/check_phpfpm_status/check_phpfpm_status
index 678170b..d0c099b 100644
--- a/check_phpfpm_status/check_phpfpm_status
+++ b/check_phpfpm_status/check_phpfpm_status
@@ -22,7 +22,7 @@ use Digest::MD5 qw(md5 md5_hex);
 
 # Nagios specific
 # Update Nagios Plugin path according to your platform/installation
-use lib "/usr/local/nagios/libexec";
+use lib "/usr/lib/nagios/plugins";
 use utils qw($TIMEOUT);
 
 # Globals
